style.css

```diff
@@ -0,0 +1,81 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+nav {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin: 3%;
+  padding: 0 10px;
+}
+
+a {
+  text-decoration: none;
+}
+
+.logo {
+  font-family: Poppins, sans-serif;
+  font-size: 18px;
+  font-weight: 700;
+  line-height: 20px;
+  letter-spacing: 0;
+  text-align: center;
+  color: #6070ff;
+}
+
+.headline {
+  width: 100%;
+  height: 718px;
+  background-image: url("./img/imagee.png");
+  background-size: 100%;
+}
```

Comment:
- [OPTIONAL] Good job for using the `background-image` 👍👍 I noticed that the background image is repeating,  can you please fix this by using `background-repeat: no-repeat;` ? ✨
- [OPTIONAL] Please consider aligning the content with `flexbox`. This gives us a lot of options to align all our elements, plus, it's responsive 📌💪